docs/get-started/create-app.md

```diff
@@ -1,13 +1,20 @@
 # {fas}`folder-plus` Create First Project
 
-Let's build a toy application with Jina. To start, use Jina CLI to make a new project:
+Let's build a toy application with Jina. To start, use Jina CLI to make a new project, we can either make a new project as a deployment or a flow: 
 
+To create a flow project with Jina CLI
 ```bash
-jina new hello-jina
+jina new hello-jina --type=flow
+```
+
+To create a deployment project with Jina CLI
+```bash
+jina new hello-jina --type=deployment
 ```
 
 This creates a new project folder called `hello-jina` with the following file structure:
 
+For a flow project: 
```

Comment:
Would be good to group Flow/Deployment file structures with tabs. Then we don't need "For a Flow project", etc.

Here's an example of the syntax (GH won't let me suggest directly via usual tool, and bc of all the backticks I can't type it directly and have it render):

![image](https://user-images.githubusercontent.com/4182659/224687470-95386b6d-549b-4218-a9a3-15f802e7471f.png)